leapp/cli/upgrade/__init__.py

```diff
@@ -201,6 +202,14 @@ def upgrade(args):
     configuration = prepare_configuration(args)
     answerfile_path = cfg.get('report', 'answerfile')
     userchoices_path = cfg.get('report', 'userchoices')
+    # report_schema None will mean that no special backwards-compatibility conversion is necessary
+    report_schema = None
+    if args.report_schema:
+        if args.report_schema < cfg.get('report', 'schema'):
```

Comment:
When I pass an argument with schema version equal to the actual one, leapp throws an error:

```
$ leapp preupgrade --report-schema 1.1.0

Error: --report-schema version can not be greater that the actual 1.1.0 one.
```

Is the error intended? The error message says that the provided schema version cannot be greater than the actual one, but that isn't the case here.